docker/docker.mk

```diff
@@ -48,13 +48,15 @@ else
   BIN_PATH := $(BIN_PATH)/debug
 endif
 
+DOCKER_IMAGE_ARCH:=amd64
 ifeq ($(TARGET_ARCH),arm)
-  DOCKER_IMAGE_PLATFORM:=$(TARGET_OS)/arm/v7
+  DOCKER_IMAGE_ARCH:=arm/v7
 else ifeq ($(TARGET_ARCH),arm64)
-  DOCKER_IMAGE_PLATFORM:=$(TARGET_OS)/arm64/v8
-else
-  DOCKER_IMAGE_PLATFORM:=$(TARGET_OS)/amd64
+  DOCKER_IMAGE_ARCH:=arm64/v8
 endif
+DOCKER_IMAGE_PLATFORM=$(TARGET_OS)/$(DOCKER_IMAGE_ARCH)
+DOCKER_BUILDX_NAME=daprbuild_$(TARGET_OS)_$(TARGET_ARCH)
+DOCKER_BUILDX_PLATFORMS=$(TARGET_OS)/arm/v7,$(TARGET_OS)/arm64/v8,$(TARGET_OS)/amd64
```

Comment:
We could have buildx build platform to be with a single variant too but adding all seems harmless.

Reply:
Reusing the same builder for all variants.